packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -409,7 +406,7 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
         page: page,
         className: css('ms-List-page'),
         key: page.key,
-        ref: page.key,
+        ref: el => (this._pageRefs[page.key] = el),
```

Comment:
I think this can be a cached method of the class to reduce reallocating a method on every render.

Reply:
I think this can be avoided entirely, updated.